service/build.gradle

```diff
@@ -31,7 +31,11 @@ dependencies {
     }
     implementation group: "com.google.guava", name:"guava"
 
-    implementation 'bio.terra:terra-common-lib'
+    // Get stairway via TCL
+    implementation "bio.terra:terra-common-lib" // 0.0.94-SNAPSHOT is defined in bio.terra.pipelines.java-common-conventions.gradle
+    implementation "org.broadinstitute.dsde.workbench:sam-client_2.13:0.1-fd8ee25"
```

Comment:
oh good catch, that was a rebasing miss. at some point something with TCL/stairway didn't work with the older sam client, but I think it was probably something that WSM was doing with authenticated users that I ended up taking out cause we don't need it. since it works with the newer version I'll keep it, but there's not a need for it per se at this time.